### ISSUE TITLE
Automation for new client configurations

### ISSUE THREAD

**Issue by esune** [bcgov/vc-authn-oidc#61]
Adding a new client to a running instance of `vc-authn` is a process that requires manually inserting rows into the database.

Creating some scripts that can help automating this process would beneficial for both new users onboarding the platform AND for the long-term maintainability of existing deployments.

Additional information on what to look for will be added to the ticket, however the service is built on top of [IdentityServer4](https://identityserver4.readthedocs.io/en/latest/) so their documentation may be useful as a reference.

**Comment by swcurran** [comment]
What about eliminating the database and making the instance stateless.  On startup, it loads it's config from file and at most puts it into an in-memory database (to preserve the existing API).  The existing API could remain (I guess) to support adding new configs, but the general model would just be to restart the server to add more configs.

**Comment by swcurran** [comment]
Just added issue #64 that we may have to support if we move ahead with SafeEntryBC. That feature will require state be saved across pods for long periods of time, so eliminating state is not possible.

As I think about it, removing state would make deploying multiple pods impossible, as the creation of the proof request, and the mobile agent requesting the URL would have to resolve to the same pod.

In short:

![image](https://user-images.githubusercontent.com/1018050/80034858-eb462580-84a3-11ea-8362-2c843c47da8c.png)

**Comment by esune** [comment]
This is resolved - at least for the purpose of our operations - by https://github.com/bcgov/a2a-trust-over-ip-configurations/pull/52.
The scripts used in the `manage` utility can be easily extracted and re-used if necessary.
